src/validations/html/rules/MismatchedTag.js

```diff
@@ -0,0 +1,139 @@
+import {List, Map, Record, Set} from 'immutable';
```

Comment:
What’s the thinking behind using `Immutable` in here rather than plain JavaScript `Object`, `Array`, `Map`, `Set`? I like `Immutable` and obviously it’s deeply baked into the Redux side of the app, but it’s not without its cost (a more verbose API, having to remember to reassign the variable every time you want to modify the collection) and it doesn’t seem like we’re gaining a lot (reassigning immutable variables vs. mutating mutable constants).